test/e2e/azure_lb.go

```diff
@@ -144,15 +146,18 @@ func AzureLBSpec(ctx context.Context, inputGetter func() AzureLBSpecInput) {
 
 		if !input.SkipCleanup {
 			By("deleting the ilb test resources")
+			Logf("deleting the ilb service: %s", ilbService.Name)
 			err = servicesClient.Delete(ctx, ilbService.Name, metav1.DeleteOptions{})
 			Expect(err).NotTo(HaveOccurred())
+			Logf("deleting the ilb job: %s", ilbJob.Name)
 			err = jobsClient.Delete(ctx, ilbJob.Name, metav1.DeleteOptions{})
 			Expect(err).NotTo(HaveOccurred())
 		}
 	}
 
 	By("creating an external Load Balancer service")
 	elbService := webDeployment.GetService(ports, deploymentBuilder.ExternalLoadbalancer)
+	Logf("starting to create an external Load Balancer service")
```

Comment:
Would it make sense to also add some logging around the time taken for the resources to be created? We can probably deduce it from other logs, just that it makes it easier for debugging.